jemallocator-global/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+//! Sets `jemalloc` as the `#[global_allocator]` on targets that support it.
+//!
+//! Just add `jemallocator-global` as a dependency:
+//!
+//! ```toml
+//! # Cargo.toml
+//! [dependencies]
+//! jemallocator-global = "0.2.0"
+//! ```
+//!
+//! and `jemalloc` will be used as the `#[global_allocator]` on targets that
+//! support it.
+//!
+//! To unconditionally set `jemalloc` as the `#[global_allocator]` enable the
+//! `force_global` cargo feature.
+
+#[cfg(jemallocator)]
```

Comment:
Not `cfg(feature = "jemallocator")`?

Reply:
With that fixed there’s a "should have an upper case name" warning.

Reply:
Yes, and with that fixed, on macos, its not working. The dependency is not enabled automatically.